doc/meetings/2017-06-26.md

```diff
@@ -3,7 +3,7 @@
 - [Github Issue](https://github.com/nodejs/LTS/issues/233)
 - [Meeting Video](https://www.youtube.com/watch?v=xSo6YiKzc5M)
 - [Previous meeting](https://github.com/nodejs/LTS/issues/225)
-- Next meeting: 17 July 2017
+- Next meeting: 19 July 2017
```

Comment:
Should be 19 September?

Reply:
Not sure what I was doing here, it should actually have been the 17th of July, see https://github.com/nodejs/Release/issues/235.

The meeting was cancelled, but we didn't know that at the time.